src/components/Modals/Snaps/SnapConfirm.tsx

```diff
@@ -62,6 +63,13 @@ export const SnapConfirm: React.FC<SnapConfirmProps> = ({ onClose }) => {
             The ShapeShift Multichain Snap is a new, experimental feature, provided "as is"
           </AlertDescription>
         </Alert>
+        <Alert status='error' borderRadius='lg' mb={4}>
+          <AlertIcon />
+          <AlertDescription>
+            Ensure you have backed up your MetaMask seed phrase. Any funds you send to non-EVM
+            chains using this Snap will be directed to the default account you set up with MetaMask.
```

Comment:
as @firebomb1 requested, lets put this in a translate and into main.json

Reply:
Pushed a commit to make that modal translatable, hope this helps. If not revert of course :D

Reply:
tyvm @firebomb1 ! Frontran me on this one, looking good 🐐